### DISCUSSION TITLE
Python code to call Fooocus to generate images?

### DISCUSSION THREAD

**Discussion by Tanoshimi** [lllyasviel/Fooocus#3438] [Q&A] [answered]
I'm working on an automated process, and I basically want to generate images in a loop from a list of prompts.  

Can someone point me in the right direction to write a python script that will do this?  I don't need the script written, just where I can find documentation or even an example.

**Comment by mashb1t** [comment, marked as answer]
Basically you have to create an AsyncTask in https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L10 and then call https://github.com/lllyasviel/Fooocus/blob/main/modules/async_worker.py#L280. All relevant information are set in the AsyncTask object.
